hail/src/main/scala/is/hail/expr/ir/Emit.scala

```diff
@@ -2378,7 +2378,7 @@ class Emit[C](
             PCanonicalTuple(true, et.emitType.storageType).constructFromFields(cb, region, FastIndexedSeq(et), deepCopy = false)
           }
 
-          val bufferSpec: BufferSpec = BufferSpec.defaultUncompressed
+          val bufferSpec: BufferSpec = BufferSpec.wireSpec
```

Comment:
Re: client crash on that test. 
Is it possible that changing to a compressed representation of a bunch of integers, actually caused it to get past the driver limitations and instead fail on the client?

Reply:
I can confirm that the difference between passing and failing test_driver_has_tiny_memory is this buffer spec here

Reply:
gonna try upping the size of the range table in `test_tiny_driver_has_tiny_memory` and restore the compression see if that works with making it oom.

Reply:
@chrisvittal I think this is what I meant to have:

```
hl.utils.range_table(50, 50).annotate(x=hl.range(100_000_000)).to_pandas()
```
Can you try this?